src/Movie-Card/Movie-Card.css

```diff
@@ -0,0 +1,6 @@
+.movie-card {
+    width: auto;
+    max-height: 100%;
+    object-fit: contain;
+    padding: 20px;
+}
```

Comment:
Not sure if this makes sense or if the cards are getting all their sizing from the defined img size in Movie-Card